peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -145,6 +145,33 @@ <h1 class="py-4">{{ document.title }}</h1>
                           </dd>
                         {% endif %}
                       {% endblock %}
+
+                      {% block document-metadata-media-summary-download %}
+                        {% if document.attachedfiles_set %}
+                          {% for attachedfile in document.attachedfiles_set.all %}
+                            {% if attachedfile.nature.name == 'Media summary' %}
+                              <dt>{% translate 'Media summary' %}</dt>
```

Comment:
No need to special case anything, just use `attachedfile.nature.name` as the `dt` value.